addons/website/static/tests/tours/edit_menus.js

```diff
@@ -260,5 +260,18 @@ wTourUtils.registerWebsitePreviewTour('edit_menus', {
         trigger: 'iframe #top_menu .nav-item:has(a.o_mega_menu_toggle:contains("Megaaaaa!")) ' +
                  '.s_mega_menu_odoo_menu',
         run: () => {}, // It's a check.
-    }
+    },
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    {
+        content: "Open nested menu item",
+        trigger: 'iframe #top_menu .nav-item:contains("Home"):not(.position-static) a.dropdown-toggle',
```

Comment:
Not sure what the `not(.position-static)` is about. Can't the selector be something simpler, like `iframe #top_menu .nav-item .dropdown-toggle`?

Reply:
Using `:nth(1)` instead.